ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -908,7 +908,7 @@ classes:
     base: [OpenstackComponent]
     properties:
       proxy_device:
-        label: Device
+        label: Proxy Device
```

Comment:
I suggest "Linux Device" since we never really seem to use the proxy word elsewhere in this zp